packages/editor/src/store/selectors.js

```diff
@@ -729,9 +729,14 @@ export const getBlocks = createSelector(
 			( clientId ) => getBlock( state, clientId )
 		);
 	},
-	( state ) => [
-		state.editor.present.blocks,
-	]
+	( state, rootClientId ) => {
+		if ( ! rootClientId ) {
```

Comment:
Given the prior noted observation about dependants caching not occurring per arguments set, I'm not sure this is a change we should want to make. Is it even relevant to the changes, or could it at least be considered separately?

Reply:
Nothing like measuring :) I couldn't find any meaningful difference between the two, so I reverted back to the simpler code as per your suggestion.